BookingProject/BookingWebAPI/AuditWebApi.Infrastructure/Kafka/KafkaConsumer.cs

```diff
@@ -0,0 +1,86 @@
+﻿using AuditWebApi.Application;
+using AuditWebApi.Application.DTOs;
+using AuditWebApi.Infrastructure.Configuration;
+using Confluent.Kafka;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace AuditWebApi.Infrastructure.Kafka
+{
+    public class KafkaConsumer : BackgroundService
+    {
+        private readonly ILogger<KafkaConsumer> _logger;
+        private readonly IConsumer<Ignore, string> _consumer;
```

Comment:
Make it generic like `Consumer<TKey, TValue>` or just `TKey`. If u use `Ignore` for consumer, that means the order of your events may be incorrect. Kafka uses message Keys to provide the correct order of consuming per partition